apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -117,31 +122,14 @@ bool MyMediaPlayer::init(int test)
 	}
 
 	switch (test) {
-	case TEST_MP3:
+	case TEST_PCM:
 		makeSource = []() {
-			auto source = std::move(unique_ptr<FileInputDataSource>(new FileInputDataSource("/rom/over_16000.mp3")));
+			auto source = std::move(unique_ptr<FileInputDataSource>(new FileInputDataSource("/rom/44100.pcm")));
+			source->setSampleRate(44100);
+			source->setChannels(2);
 			source->setPcmFormat(AUDIO_FORMAT_TYPE_S16_LE);
 			return std::move(source);
 		};
-		break;
```

Comment:
we added it to test below types before.

Reply:
all files under /rom/* will be listed out when selecting source.
and then user can select any of them to test, may be a mp3/aac/mp4/opus/... postfix file.
e.g.
```txt
====================
 0. Exit APP
 1. Test PCM
 2. Test BUFFER
 3. Test HTTP
 4. /rom/play.mp4
 5. /rom/mono_44_1k.mp3
 6. /rom/mono_24k.mp3
 7. /rom/res_16k.mp3
====================
```